### DISCUSSION TITLE
Chia Wallet. Question

### DISCUSSION THREAD

**Discussion by San1k1983** [Chia-Network/chia-blockchain#13596] [Support]
Good afternoon.

I don't see XCH coins in the official wallet.
I use the latest version of the software.
ver ChoaBlockchain 1.6.1b3-dev43.
it's been at least 2 hours.

What could be the problem?

--
best regards,
Alexander Shv.

**Comment by brause** [comment]
If your xch does not show up you might not be fully synced. What is your sync status ?
